### PR TITLE
Update production docs

### PR DESCRIPTION
**Pull request by kevinlul**

### CODE REVIEW

File: docs/production.md [hunk 1]
```diff
@@ -19,32 +18,49 @@ source code is no longer mounted but baked into the image.
...
-## Running
+This deployment serves [Care4Rare](http://care4rare.ca/) as part of [Genomics4RD](https://www.genomics4rd.ca/).
+It runs on [HPC4Health](http://hpc4health.ca/) infrastructure in a dedicated tenancy referred to as
+the CHEO-RI tenancy, or CHEO_G4RD.
 
-We use an alternate Compose configuration to add nginx, run Gunicorn, and stop forwarding container
-ports to the host unless explicitly required outside of development. The specific secret credentials
-for this deployment must be in `.env` as in development.
+Stager is deployed as a Docker container running Gunicorn per [docker-compose.cheo.yaml](/docker-compose.cheo.yaml).
+Other needed services like MySQL and MinIO are separately already running in the tenancy. A reverse proxy for the entire
```
Comment (hannahle) [inline docs/production.md:49]: Maybe we can link to the appropriate nginx config file as well?

**Review by hannahle — COMMENTED** [review]
- It would also be really helpful to see a diagram of the different hosts/ virtual machines and how they relate to each other. 
- I used to make [this diagram](https://lucid.app/lucidchart/d5a719af-7a6e-48a9-a450-f280d5875d99/edit?viewport_loc=-717%2C-263%2C2969%2C1512%2C0_0&invitationId=inv_7b7fd244-ff19-4748-a28e-0ee98db9eb13) for OSMP when we were outlining how the production stack should look. (We don't currently go by this diagram anymore but you get the idea).
- At a high level, what makes production still slightly confusing to me it seems like there are multiple VMs, but I don't know exactly what or where they are, and how to access them. More specific commands would also be helpful.

**Comment by delvinso** [comment]
Looks good to me! Just wondering if there are any diagrams that could be put in here, specifically for nginx/gunicorn and github actions. I think in these cases having a diagram could really help supplement the information here!

**Comment by delvinso** [comment]
> * It would also be really helpful to see a diagram of the different hosts/ virtual machines and how they relate to each other.
> * I used to make [this diagram](https://lucid.app/lucidchart/d5a719af-7a6e-48a9-a450-f280d5875d99/edit?viewport_loc=-717%2C-263%2C2969%2C1512%2C0_0&invitationId=inv_7b7fd244-ff19-4748-a28e-0ee98db9eb13) for OSMP when we were outlining how the production stack should look. (We don't currently go by this diagram anymore but you get the idea).
> * At a high level, what makes production still slightly confusing to me it seems like there are multiple VMs, but I don't know exactly what or where they are, and how to access them. More specific commands would also be helpful.

For your last point my understanding is a lot of this information is contained in a private repo, at least for CHEO-RI, and on a 'need to know' basis (as in I'm not sure if it's good practice for _everyone_ to have access to the prod services but I will let Kevin speak more to this)

**Comment by hannahle** [comment]
> > * It would also be really helpful to see a diagram of the different hosts/ virtual machines and how they relate to each other.
> > * I used to make [this diagram](https://lucid.app/lucidchart/d5a719af-7a6e-48a9-a450-f280d5875d99/edit?viewport_loc=-717%2C-263%2C2969%2C1512%2C0_0&invitationId=inv_7b7fd244-ff19-4748-a28e-0ee98db9eb13) for OSMP when we were outlining how the production stack should look. (We don't currently go by this diagram anymore but you get the idea).
> > * At a high level, what makes production still slightly confusing to me it seems like there are multiple VMs, but I don't know exactly what or where they are, and how to access them. More specific commands would also be helpful.
> 
> For your last point my understanding is a lot of this information is contained in a private repo, at least for CHEO-RI, and on a 'need to know' basis (as in I'm not sure if it's good practice for _everyone_ to have access to the prod services but I will let Kevin speak more to this)

That's a really fair point! Especially since our repo is public. In that case, a diagram would suffice of how different VMs interact would be sufficient.

**Comment by kevinlul** [comment]
I'll draw them up or find Pouria's slides, but they can't go here.

**Comment by jennyziyi-xu** [comment]
Here are some points that I don't know if we should mention because they might sound really specific and it might not be appropriate to disclose them. 

- Is Nginx a transparent, anonymous, or distorting proxy? Users might need to know this information to gauge the level of security of the network in handling sensitive data.
- Nginx also is a load balancer and a web cache. 
- What cookies are created by the web server and placed on the user's computer. 
- The specifications of the production server and MinIO server: OS, RAM, CPUs, storage, acceleration if any

**Pull request closed without merging**